added a share button to the start activity - Fix Issue #1388

justdvnsh opened the pull request:
added a share button to the start activity, added a share intent to share the link of the apk and also added all the dimentions of the app in the dimen.xml

### Description
Fixed issue #1388

Fixes # [1388]

### Type of Change:
**Delete irrelevant options.**

- Code
- User Interface

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] Any dependent changes have been merged 

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [x] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

justdvnsh committed:
added a share button to the start activity and added the share intent to it. Also added the dimentions in the dimens.xml

aditmehta9 commented:
> 
> 
> added a share button to the start activity, added a share intent to share the link of the apk and also added all the dimentions of the app in the dimen.xml
> ### Description
> 
> Fixed issue #1388
> 
> Fixes # [1388]
> ### Type of Change:
> 
> **Delete irrelevant options.**
> 
>     * Code
> 
>     * User Interface
> 
> 
> **Code/Quality Assurance Only**
> 
>     * Bug fix (non-breaking change which fixes an issue)
> 
>     * New feature (non-breaking change which adds functionality pre-approved by mentors)
> 
> 
> ### Checklist:
> 
> **Delete irrelevant options.**
> 
>     * [x]  My PR follows the style guidelines of this project
> 
>     * [x]  I have performed a self-review of my own code or materials
> 
>     * [x]  I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
> 
>     * [x]  Any dependent changes have been merged
> 
> 
> **Code/Quality Assurance Only**
> 
>     * [x]  My changes generate no new warnings
> 
>     * [x]  My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
> 
>     * [x]  I have added tests that prove my fix is effective or that my feature works
> 
>     * [ ]  New and existing unit tests pass locally with my changes
> 
>     * [ ]  Any dependent changes have been published in downstream modules

Hey, Divyansh I have been waiting to be assigned by mentors for this issue. In many issues, students are asking to get assigned by a mentor. You have to first ask a mentor to work on that issue. See in issue I have asked. Please follow the contribution guidelines.

justdvnsh commented:
> > added a share button to the start activity, added a share intent to share the link of the apk and also added all the dimentions of the app in the dimen.xml
> > ### Description
> > Fixed issue #1388
> > Fixes # [1388]
> > ### Type of Change:
> > **Delete irrelevant options.**
> > ```
> > * Code
> > 
> > * User Interface
> > ```
> > 
> > 
> > **Code/Quality Assurance Only**
> > ```
> > * Bug fix (non-breaking change which fixes an issue)
> > 
> > * New feature (non-breaking change which adds functionality pre-approved by mentors)
> > ```
> > 
> > 
> > ### Checklist:
> > **Delete irrelevant options.**
> > ```
> > * [x]  My PR follows the style guidelines of this project
> > 
> > * [x]  I have performed a self-review of my own code or materials
> > 
> > * [x]  I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
> > 
> > * [x]  Any dependent changes have been merged
> > ```
> > 
> > 
> > **Code/Quality Assurance Only**
> > ```
> > * [x]  My changes generate no new warnings
> > 
> > * [x]  My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
> > 
> > * [x]  I have added tests that prove my fix is effective or that my feature works
> > 
> > * [ ]  New and existing unit tests pass locally with my changes
> > 
> > * [ ]  Any dependent changes have been published in downstream modules
> > ```
> 
> Hey, Divyansh I have been waiting to be assigned by mentors for this issue. In many issues, students are asking to get assigned by a mentor. You have to first ask a mentor to work on that issue. See in issue I have asked. Please follow the contribution guidelines.

Hey. Yeah. Sorry, I missed the contribution guidelines. So you have already claimed the issue, so can I also claim it ?

aditmehta9 commented:
Maybe not because I asked first. Mentors will assign who will do. Right now mentors are busy because of GSoC.
